.github/workflows/publish.yaml

```diff
@@ -236,12 +236,27 @@ jobs:
         with:
           name: '${{ needs.build-tauri.outputs.channel }}-${{ matrix.platform }}-${{ github.run_number }}'
           path: release
+      - name: Set file as a variable
+        id: set-path
+        run: |
+          VALUE=$(find release -name "*.msi" -type f -printf '%P\n')
+          echo "::set-output name=msi_file::$VALUE"
```

Comment:
Is this syntax still supported? I thought it was:

```
- name: Set output
  run: echo "{name}={value}" >> $GITHUB_OUTPUT
```

Reply:
Old school.

Technically they are still supported, but yes, I will change them to the new format.